manifests/vhost.pp

```diff
@@ -2360,6 +2360,12 @@
     }
   }
 
+  if ($ssl) {
+    check_http { "https://${servername}": }
+  } else {
+    check_http { "http://${servername}": }
```

Comment:
You should take the port(s) into account as well.

Reply:
does this look like I'm finished yet? :rofl: